sdk/keyvault/azure-keyvault-certificates/tests/certificates_async_preparer.py

```diff
@@ -28,10 +28,22 @@ class AiohttpTestTransport(AioHttpTransport):
         return response
 
 
+class NoSleepTransport(AiohttpTestTransport):
+    async def sleep(self, _):
+        """Prevent sleeping between retries. (RetryPolicy doesn't expose configuration to ignore Retry-After headers.)
+        """
+        f = asyncio.Future()
```

Comment:
A simple return should be enough, I don't understand why you make this complicated? Nothing forces an async method to have any await or asyncio kind of code.
At worst if you really want to write something, I would do `asyncio.wait(0)` anyway

Reply:
That's a good point, thank you, I've simplified it.